marimo/_config/config.py

```diff
@@ -291,6 +308,7 @@ class PartialMarimoConfig(TypedDict, total=False):
         "auto_instantiate": True,
         "auto_reload": "off",
         "on_cell_change": "autorun",
+        "output_max_byes": 5_000_000,
```

Comment:
The config should be backward compatible with the environment variable, at least for now: so if the config is not set, we should check if MARIMO_OUTPUT_MAX_BYTES is defined.